### PR TITLE
Create runGist helper; minimal gist acceptance test

### PR DESCRIPTION
**Pull request by Gaurav0**
Relevant to issue #84

### CODE REVIEW

**Comment by joostdevries** [comment]
I actually like this a lot :-)

**Comment by joostdevries** [comment]
@rwjblue what do you think?

File: tests/helpers/start-app.js [hunk 1]
```diff
@@ -10,6 +11,7 @@ export default function startApp(attrs) {
 
   Ember.run(function() {
     application = Application.create(attributes);
+    Ember.Test.registerAsyncHelper('runGist', runGist);
```
Comment (rwjblue) [inline tests/helpers/start-app.js]: This only needs to happen once, when the module is loaded.  Including it here (inside the `startApp` function) will have this ran once for each acceptance test.

I prefer to have the `run-gist` helper imported and registered in `tests/test-helper.js`.  Any objections?

File: tests/acceptance/routing-test.js [hunk 2]
```diff
@@ -15,87 +15,51 @@ module('Acceptance | routing', {
...
-        return resolve();
-      });
-    });
+    iframe_window.click(iframe_window.find(aboutLink));
   });
 
   andThen(function() {
-    iframe_window.visit('/');
-  });
+    assert.equal(iframe_window.find(outletText).text().trim(), 'About Page', 'About Link leads to About Page being displayed');
```
Comment (rwjblue) [inline tests/acceptance/routing-test.js]: This is repeated a few times, can it be extracted to a test helper?

Something like:

```javascript
const iframe = '#dummy-content-iframe';

Ember.Test.registerHelper('outputPane', function(app) {
  return find(iframe)[0].contentWindow;
});

Ember.Test.registerHelper('outputContents', function(app, selector) {
  let output = app.testHelpers.outputPane();
  return output.find(selector).text().trim();
});

// then you can use them as follows:
outputPane().click(aboutLink);
andThen(() => {
  assert.equal(outputContents(outletText), 'About Page');
});
```

**Comment by Gaurav0** [comment]
@joostdevries @rwjblue Updated with suggested changes.

**Comment by rwjblue** [comment]
Awesome!  Thank you!

**Pull request merged**